code/modules/reagents/Chemistry-Recipes_vr.dm

```diff
@@ -85,6 +85,15 @@
     required_reagents = list("water" = 1)
     catalysts = list("fluorine" = 10)
     result_amount = 1
+	
+/datum/chemical_reaction/firefightingfoamqol //Please don't abuse this and make us remove it. Seriously.
+    name = "Firefighting Foam EZ"
+    id = "firefighting foam ez"
+    result = "firefoam"
+    required_reagents = list("water" = 1)
+    catalysts = list("firefoam" = 0.01)
+	inhibitors = list("fluorine" = 0.01)
```

Comment:
your problem's here, needs a tab

Reply:
I don't follow. Where is the tab needed?

Reply:
You either need to replace all of the spaces here with single tabs, or change this line to four spaces instead of a tab
```suggestion
    inhibitors = list("fluorine" = 0.01)
```